.github/workflows/CI.yml

```diff
@@ -64,6 +64,7 @@ jobs:
         with:
           files: |
             production_configs/*/*.yml
```

Comment:
why don't we just use `*.yaml` extension, and therefore we change all the `*.yml` that we are using

Reply:
we do use `.yaml` [as recommended ](https://yaml.org/faq.html)
Better look for both just in case...